guide/matrix-vs-telegram.md

```diff
@@ -28,27 +28,27 @@ Despite recent studies, Telegram's use of its own MTProto protocol remains a deb
 | Attachments | **Maximum 2 GB.** | Maximum 50~100 MB (for most homeservers; customizable if you run your own homeserver). |
 | Reactions | Very limited. Must be enabled by group admins in groups. | All unicode emotes and [text](../features/#reactions). |
 | Stickers | Up to 200 packs of 120 static or 50 animated each. | **Unlimited (static or animated) with setup.** See [here](../features/#stickers). |
-| Read receipts | Supported ambiguously. | **Supported.** |
+| Public read receipts | Supported ambiguously. | **Supported.** |
 | **Direct messages** | Not encrypted unless explicitly opted into secret chat, which cannot be carried across devices. VoIP is encrypted. | **Encrypted by default**, including VoIP. |
 | **Group chats** | You can join up to 500 groups and channels. | You can join an **unlimited** amount of rooms. |
 | VoIP in groups | Supported. | Not supported until early 2022. |
 | Organizing chats | You can pin or archive groups (similar to favourite and low priority on Element/SchildiChat). | Rooms can be included within an unlimited amount of Spaces. Spaces may also include other Spaces. |
 | Group chat privacy | You may deny new members from reading more than 100 messages prior to them joining. | You may deny new members from reading messages prior to them being invited / joining. You may also allow or deny guest access (such as [Matrix Static](http://view.matrix.org/)) from reading messages. You may also enable encryption[^1]. |
-| Publicity | Any group or channel set to public can be listed in search results, but how they are shown is arbitrary, as global search is not always visible to users. | Each homeserver has a room directory which anyone may publish to. |
+| Publicity | Any group or channel set to public can be listed in search results, but how they are shown is arbitrary, as global search is not always visible to users. | Each homeserver has a room directory which anyone in that homeserver may publish to. |
 | Invite | Through directly inviting users, or through generating invite links. | Through directly inviting users, or through shareable [addresses](../features/#promotion). |
 | Permissions | Permissions of each administrator are set manually. All admins are equal (except owner). | 2^54 power levels (I think it's -2^53 to 2^53-1, however I highly doubt you will *ever* reach that limit). A user acquires a permission if their power level is equal to or higher than the power level required for the specific permission. |
-| Ads | Popular channels now carry ads that you cannot opt out. | **It is not possible for Matrix to implement ads.** |
+| Ads | Popular channels now carry ads that you cannot opt out. | **Most Matrix home servers do not allow ads in the chats hosted in them. No known Matrix home server inserts ads into room state.** |
```

Comment:
I guess it is possible for a homeserver to specifically implement ads, but then users can still opt out pretty easily (eg. ACL). Servers can also manipulate events to remove ads. So I guess it is not possible to *effectively* implement ads?

Reply:
There are actually more sinister ways such as appearing as another room member from your home server and sending ads as them, while not routing that ad-injection message to the user impersonated (a home server admin might easily impersonate a local user with the current event signing scheme).

Reply:
So homeservers doing that can be ACL'd?

Reply:
I think the point is more that matrix is not immune to this attack and yes ofc you can just ACL ban those homeservers from your room to make it so a homeserver you dont agree with is ACL banned if your in control of a room.

Reply:
```suggestion
| Ads | Popular channels now carry ads that you cannot opt out. | It is technically possible for a homeserver to insert ads, but **there are no known occurrences**. |
```